browse.html

```diff
@@ -166,20 +166,15 @@ <h1>Hold onto your ava...</h1>
         const div = document.createElement('div');
         div.classList.add('token');
         const href = `inspect.html?p=${name}`;
-        const size = 300;
+        const size = '100%';
```

Comment:
Are sizes always going to be the same for all screenshots? I think if a user uploads a custom shot, this might break the design.

Reply:
I'm not sure if they'd always be the same but I just tried to put a random 1000x1000 image in and it seems to work:

![image](https://user-images.githubusercontent.com/8850830/87940905-31eef000-ca92-11ea-8408-cf1f2c9abc42.png)

each `.token` element should also have its height/width constrained by the `grid` so I don't _think_ it should be able to escape those bounds!

Reply:
Perfect, thanks for checking!